package.json

```diff
@@ -111,6 +112,7 @@
     "uuid": "^3.0.1",
     "which": "^1.3.0",
     "wordwrap": "^1.0.0",
-    "xml2js": "^0.4.19"
+    "xml2js": "^0.4.19",
+    "superagent": "^1.7.2"
```

Comment:
We already have request in the dependency list, why did you drag in yet another http client library?

Reply:
Make sense, thanks for pointing!